src/server-extension/query/tokenEntityByOwner.ts

```diff
@@ -0,0 +1,48 @@
+export const tokenEntityByOwner = `WITH CheapestNFT AS (
+    SELECT 
+        ne.token_id,
+        MIN(ne.price) AS "cheapestNFTPrice"
+    FROM 
+        nft_entity ne
+    WHERE 
+        ne.current_owner = $1
+    GROUP BY
+        ne.token_id
+)
+
+SELECT
+    t.id as id,
+    t.name as name,
+    t.image as image,
+    t.media as media,
+    t.metadata as metadata,
+    me.id as "metaId",
+    me.description as "metaDescription",
+    me.image as "metaImage",
+    me.animation_url as "metaAnimationUrl",
+    t.block_number as "blockNumber",
```

Comment:
any camelcase is ignored by resolver

Reply:
it actually isn't when you put it in " "
I wanted to keep the names of fields as they are on TokenEntity

![image](https://github.com/kodadot/stick/assets/22791238/b1ee6af6-388c-4c99-98e8-6d1f1e2f6b35)